client/state/ui/guided-tours/actions.js

```diff
@@ -49,3 +49,7 @@ function addSeenGuidedTour( getState, tourName, finished = false ) {
 		}
 	] );
 }
+
+export function resetGuidedToursHistory() {
+	return savePreference( 'guided-tours-history', [] );
```

Comment:
I think any solution beyond what's here would be overkill, but semantically `resetGuidedToursHistory` would be changing the value back to some original value... if for some reason we started with some other initial value we'd  no longer be 'resetting' as such.

This can be ignored though, since we don't have an initial value for `guided-tours-history` - it's mostly just me thinking out loud :)